google/cloud/storage/examples/storage_otel_samples.cc

```diff
@@ -0,0 +1,123 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/internal/port_platform.h"
+#if GOOGLE_CLOUD_CPP_STORAGE_HAVE_GRPC && GOOGLE_CLOUD_CPP_HAVE_COROUTINES
+// The final blank line in this section separates the includes from the function
+// in the final rendering.
+//! [instrumented-client-includes]
+#include "google/cloud/opentelemetry/configure_basic_tracing.h"
+#include "google/cloud/storage/async_client.h"
+#include "google/cloud/opentelemetry_options.h"
+
+//! [instrumented-client-includes]
+#include "google/cloud/storage/examples/storage_examples_common.h"
+#include "google/cloud/internal/getenv.h"
+#include <cstdint>
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+
+namespace {
+namespace examples = ::google::cloud::storage::examples;
+
+void InstrumentedClient(std::vector<std::string> const& argv) {
+  if (argv.size() != 3) {
+    throw examples::Usage{
+        "instrumented-client <project-id>"
+        " <bucket-name> <object-name>"};
```

Comment:
nit: this usage string looks like a one-liner

Reply:
Fixed.